.appveyor.yml

```diff
@@ -0,0 +1,10 @@
+install:
+  - pip install tox
+build: false
+test_script:
+  - tox -e "py{27,36}"
+matrix:
+  fast_finish: true
+branches:
```

Comment:
Please no.  I do development on branches on my fork, and this requires that I remove this every time I want to get involved. (and I usually spend 10 minutes trying to work out why the build didnt start.)

You can do the same now that this repo isnt in your userspace, or you can do `[skip ci]`.

Reply:
Hmm, that's a good point. I think I saw an option for it to not do branch builds when there are PRs. That's the reason I add it: to avoid double-builds on pull requests on the same repo.

Reply:
For myself, I like opening PRs early, so the master-only restriction doesn't bother me, but I can see that would be annoying if you want to check on the tests first. Is this something that you think we should also change for the Travis config?

Reply:
The .travis.yml does not have this nonsense.

Reply:
> You can do the same now that this repo isnt in your userspace

That's actually a challenge. If I fork it, I'll break the redirection that GitHub added when the repository was moved.

Reply:
> The .travis.yml does not have this nonsense.

It does, I'm just using the new `if` condition instead of this spelling, which is what I've been using.

https://github.com/tox-dev/tox-travis/blob/master/.travis.yml#L29

Reply:
ah true, I remember having to workaround this problem previously, making my development time longer.